openapi/openapi.bzl

```diff
@@ -141,7 +141,7 @@ openapi_gen = rule(
         ),
     },
     outputs = {
-        "codegen": "%{name}_codegen.srcjar",
+        "codegen": "%{name}_codegen.src.jar", # IntelliJ does not understand .srcjar
```

Comment:
Have you pulled this down into chapstick and tested the build with it (is there an accompanying chapstick PR)? The reason I ask is before we've had issues with how bazel treats .src.jar files (see https://github.com/meetup/meetup/blob/master/modules/entities/entities.bzl#L86). But I'm struggling to remember the finer details besides "it didn't work"

Reply:
I see. Then this hack above ^^ should be used instead of changing the rules themselves.

Reply:
I wish it was in a more visible place.